### PR TITLE
Added molecule test for the Wazuh Agent

### PR DESCRIPTION
**Pull request by dj-wasabi**
This PR contains the following changes:
* Added a Molecule scenario for testing the Wazuh Agent role.
* Added various minor changes that where found during linting;

**Scenario**
A Molecule scenario `wazuh-agent` is added and starts a single Wazuh Manager on CentOS 7. Once that is done, it start 5 containers (CentOS 6+7, Ubuntu 14, 16 + 18) and install the Wazuh Agent. During the `molecule verify`, a test is executed on the Wazuh Manager to see if all agents are registered correctly.

**linting**
During executing `molecule lint -s wazuh-agent`, a lot of messages like the following where printed:

```
      71:3      error    wrong indentation: expected 4 but found 2  (indentation)
      75:21     warning  truthy value should be true or false  (truthy)
      298:247   error    trailing spaces  (trailing-spaces).
      222:6     warning  missing starting space in comment  (comments)
```

I had to change various file to make these lint issues disappear. Most of them where indentation errors, 1 or more spaces on the end of the line. Most of them where changing `no` or `yes` to `false` or `true`. In some of the playbooks, I had to add `---` to the beginning of the file, so it is correctly `yaml`.

If there are any questions, don't hesitate to ask. I did not knew that the PR would be that big (58 files changed), so apologies for that.

### CODE REVIEW

**Comment by manuasir** [comment]
Hi @dj-wasabi ,

As always, you did an amazing and valuable job again. Much appreciated.

Regards

File: roles/wazuh/ansible-wazuh-manager/tasks/RedHat.yml [hunk 1]
```diff
@@ -16,15 +16,15 @@
...
+    gpgcheck: true
   when: ansible_distribution == 'Fedora'
 
 - name: AmazonLinux | Get Nodejs
   shell: |
     set -o pipefail
     curl --silent --location https://rpm.nodesource.com/setup_8.x | bash -
   args:
-    warn: no
+    warn: fakse
```
Comment (manuasir) [inline roles/wazuh/ansible-wazuh-manager/tasks/RedHat.yml]: Hi @dj-wasabi , I've noticed that there's a typo here.

Comment (dj-wasabi) [reply roles/wazuh/ansible-wazuh-manager/tasks/RedHat.yml]: Thanks, good catch! Pushed the fix.

**Comment by dj-wasabi** [comment]
Anytime! @manuasir  😄

**Comment by dj-wasabi** [comment]
Thank you for merging! 👍 
Already working on the next one! 😄

**Pull request merged**